spansy/src/http/types.rs

```diff
@@ -370,6 +436,12 @@ impl Spanned for Body {
     }
 }
 
+impl ToRangeSet<usize> for Body {
+    fn to_range_set(&self) -> RangeSet<usize> {
+        self.span.indices.clone()
+    }
+}
+
 /// An HTTP request or response body content.
```

Comment:
RFC calls the original body `payload body` but after chunking is applied, it is called `message body`. Let's be explicit here.
```suggestion
/// An HTTP request or response payload body content.
```

Reply:
This was updated in #21